e2e/test_operator/operator_test.go

```diff
@@ -1309,4 +1309,54 @@ var _ = Describe("Operator", Label("e2e", "pr"), func() {
 			})
 		})
 	})
+
+	// This test is pending, as all Pods will be restarted at the same time, which will lead to unavailability without
+	// using DNS.
+	PWhen("crash looping the sidecar for all Pods", func() {
```

Comment:
The test looks good, but is it actually testing the CRD (or is that even possible)? There are plenty of tests from what I was seeing that check the behviour of this field, but none of them check that the CRD will reject the value `*`.

Reply:
It does test the CRD, as we are setting the `*`, if the CRD doesn't allow this the test would fail. The problem with this test is, that the cluster will be broken after this. I will create a follow up issue to fix this.